.github/workflows/msbuild-and-test.yml

```diff
@@ -0,0 +1,97 @@
+name: Build and Test Orchard Core solution
```

Comment:
The name needs to be adjusted.

Reply:
Still 😉

Reply:
I guess you're looking at an older version because I fixed this: https://github.com/Lombiq/GitHub-Actions/commit/bc261f7f479c5847fca86b2b5c788be39c1fb40f